R/impact_method2.R

```diff
@@ -15,8 +15,8 @@
 ##' impact outcome can be provided as age specific if simplified=FALSE
 ##' @export
 impact_calculation <- function(con, meta, year_min = 2000, year_max = 2030, routine_tot_rate_shape = "trace_cohort", method = "method2") {
-
-if(nrow(meta) == 0L) stop("No active recipe found in 'meta'!")
+  
```

Comment:
your editor seems to have introduced a lot of whitespace changes - can you try and remove this please?

Reply:
Done.